src/components/Detail.jsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import { NavLink } from 'react-router-dom';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faCircleLeft } from '@fortawesome/free-regular-svg-icons';
+
+const Detail = () => {
+  const coin = useSelector((store) => store.detail);
+
```

Comment:
- [ ] [Optional]Fantastic design and nice Implementation @Retky , currently when I refresh your detail page, your data gets wiped off and this is not good for the user experience, you can make your data stay by just adding some codes. Kindly check the picture below to see how I added it to your code :heart: 
![Screenshot from 2022-05-19 16-39-20](https://user-images.githubusercontent.com/66097371/169341642-4b0bff2d-fd88-40d2-8d4a-77f9f0069130.png)

Reply:
Thanks, this is extremely helpful 😃; I'll implement it.